internal/process/steps/lifecycle_manager.go

```diff
@@ -54,6 +54,10 @@ func KymaName(operation internal.Operation) string {
 	return strings.ToLower(operation.RuntimeID)
 }
 
+func KymaRuntimeResourceName(operation internal.Operation) string {
```

Comment:
Where is this used?

Reply:
We decided to create single point of truth for Runtime Resource as well. I will add invocation in next commit.

Reply:
Initially I wanted to add invocation in separate PR, but it is more consistent if I add it here.